contract/src/main/kotlin/com/ritense/valtimo/contract/validation/autoconfigure/ValidatorAutoConfiguration.kt

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2015-2022 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ritense.valtimo.contract.validation.autoconfigure
+
+import com.ritense.valtimo.contract.validation.listener.ValidatorReadyEventListener
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean
+import org.springframework.context.ApplicationContext
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import javax.validation.Validation
+import javax.validation.Validator
+
+@Configuration
+class ValidatorAutoConfiguration {
+
+    @Bean
+    @ConditionalOnMissingBean(Validator::class)
+    fun validator(): Validator {
```

Comment:
I don't think this has to be a bean, can just initialize it in the companion class or the onApplicationReady method for the ValidatorReadyEventListener